Camera2Basic/app/src/main/java/com/example/android/camera2/basic/fragments/CameraFragment.kt

```diff
@@ -73,10 +73,7 @@ import kotlin.coroutines.suspendCoroutine
 
 class CameraFragment : Fragment() {
 
-    /** Android ViewBinding */
-    private var _fragmentCameraBinding: FragmentCameraBinding? = null
-
-    private val fragmentCameraBinding get() = _fragmentCameraBinding!!
+    private lateinit var fragmentCameraBinding: FragmentCameraBinding
```

Comment:
this patch is not needed: could not use lateinit type: could you revert this patch to its previous one?

That would align with the recommendation: https://developer.android.com/topic/libraries/view-binding#fragments

thank you!

Reply:
Done!